setup.py

```diff
@@ -13,8 +13,12 @@
     install_requires=requirements,
     author='Vince Knight, Owen Campbell, Karol Langner, Marc Harper',
     author_email=('axelrod-python@googlegroups.com'),
-    packages=['axelrod', 'axelrod.strategies', 'axelrod.tests'],
+    packages=['axelrod', 'axelrod.strategies', 'axelrod.tests', 'axelrod.data'],
     url='http://axelrod.readthedocs.org/',
     license='The MIT License (MIT)',
     description='Reproduce the Axelrod iterated prisoners dilemma tournament',
+    include_package_data=True,
```

Comment:
I don't think travis specifically tests this (just fyi) but appveyor does test the setup install (because we had some windows problems with that at some point...).